book/book.json

```diff
@@ -1,4 +1,5 @@
 {
+  "gitbook": "2.x.x",
```

Comment:
Yep, I agree, this should be in its own PR. 

The issue here is that the project has never specified a gitbook version, so the build has always been picking up whatever the latest gitbook release is... so the build tools have been in constant flux with no visibility or oversight. I believe the least intrusive and safest change is actually to "freeze" the gitbook version at whatever the last successful build is (2.7.6 I think).

Before I open a different PR, are you ok with this approach?